src/containers/SignUpWithFormik/index.jsx

```diff
@@ -0,0 +1,197 @@
+import React, { useState, useEffect, useRef } from 'react'
+import { Formik } from 'formik'
+import * as Yup from 'yup'
+import Button from '@material-ui/core/Button'
+import CssBaseline from '@material-ui/core/CssBaseline'
+import TextField from '@material-ui/core/TextField'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import Checkbox from '@material-ui/core/Checkbox'
+import Link from '@material-ui/core/Link'
+import Grid from '@material-ui/core/Grid'
+import Box from '@material-ui/core/Box'
+import InputLabel from '@material-ui/core/InputLabel'
+import Select from '@material-ui/core/Select'
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined'
+import Typography from '@material-ui/core/Typography'
+import Container from '@material-ui/core/Container'
+import { FormContainerStyled, FormStyled, FormControlStyled, AvatarStyled, ErrorStyled } from './style'
+
+const validationSchema = Yup.object().shape({
+  firstName: Yup.string()
+    .min(2, 'Too Short!')
+    .max(50, 'Too Long!')
+    .required('Required'),
+  lastName: Yup.string()
+    .min(2, 'Too Short!')
+    .max(50, 'Too Long!')
+    .required('Required'),
+  email: Yup.string()
+    .email('Invalid email')
+    .required('Required'),
+  password: Yup.string()
+    .min(2, 'Too Short!')
+    .max(8, 'Too Long!')
+    .required('Required'),
+  age: Yup.number().required('Required')
+})
+
+const initialValues = {
+  firstName: '',
+  lastName: '',
+  email: '',
+  password: '',
+  age: '',
+  allowExtraEmails: false
+}
+
+function SignUpWithFormik() {
+  const inputLabel = useRef(null);
+  const [labelWidth, setLabelWidth] = useState(0);
```

Comment:
![labelwidth](https://user-images.githubusercontent.com/4191549/66238273-92f93300-e6cd-11e9-9d5f-006215c5c388.png)

There is an error because of that variable being used in HTML.